src/pkg/cli/debug.go

```diff
@@ -74,15 +81,59 @@ func Debug(ctx context.Context, c client.FabricClient, p client.Provider, etag t
 		Services: failedServices,
 		Project:  project.Name,
 	}
-	err := p.Query(ctx, &req)
+
+	if etag != "" {
+		err := p.Query(ctx, &req)
+		if err != nil {
+			return err
+		}
+	} else {
+		return errors.New("no information to use for debugger")
+	}
```

Comment:
Nit pick, but you can flatten this
```suggestion
	if etag == "" {
		return errors.New("no information to use for debugger")
	}

	err := p.Query(ctx, &req)
	if err != nil {
		return err
	}
```